docs/data/date-pickers/date-picker/LocalizedDatePicker.tsx

```
@@ -17,15 +17,8 @@ const localeMap = {
   de: deLocale,
 };
 
-const maskMap = {
-  fr: '__/__/____',
-  en: '__/__/____',
-  ru: '__.__.____',
-  de: '__.__.____',
-};
```

Since default mask is inferred, this can be done